<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>loading Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>stop loading Button</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>loading Combobox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>stop loading Combobox</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>loading Label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>stop loading Label</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>loading ListWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>stop loading ListWidget</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>31</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>32</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>loading TableWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>stop loading TableWidget</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>loading Widget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>stop loading Widget</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_17">
       <property name="text">
        <string>loading all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_18">
       <property name="text">
        <string>stop loading all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_4</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>962</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_7</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>933</x>
     <y>610</y>
    </hint>
    <hint type="destinationlabel">
     <x>899</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_9</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>971</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_11</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>920</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_13</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>933</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_5</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1078</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>1061</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_8</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1091</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>1084</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_10</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1106</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>1102</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_12</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1014</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>1020</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_14</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1106</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>1106</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_15</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>895</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>890</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_18</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_16</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1003</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>995</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
